src/services.js

```diff
@@ -8,8 +8,41 @@ const services = {
    */
   running: [],
   /**
+   * @property {array} All services disabled by instance task
+   */
+  disabled: [],
+  /**
+   * Checks if given task has a 'disable' key, and removes services accordingly
+   * @param {Object} cfg Instance config object
+   * @returns {Object} formatted config object
+   */
+  checkDisabled: (cfg) => {
+    // TODO: chained command edge cases:
```

Comment:
The best way I can think to handle chained tasks is to build a separate method that you run each task through. It builds an object with the service name as the key and increments itself so you get a count of the disabled services. Then you just go down the object and if the count is equal to the number of tasks then you add it to the `disabled` array, otherwise at least one of the tasks requires it.